app/screens/home/channel_list/additional_tablet_view/additional_tablet_view.tsx

```diff
@@ -34,12 +35,13 @@ const AdditionalTabletView = ({onTeam, currentChannelId, isCRTEnabled}: Props) =
     const [initiaLoad, setInitialLoad] = useState(true);
 
     useEffect(() => {
-        const listener = DeviceEventEmitter.addListener(Navigation.NAVIGATION_HOME, (id: string) => {
+        const listener = DeviceEventEmitter.addListener(Navigation.NAVIGATION_HOME, (id: string, params?: unknown) => {
```

Comment:
New behavior, we should add (or modify) tests to count for the new behavior.

Reply:
This component has a missing test and is not a part of the scheduled post. It can be taken up later.

Reply:
@Rajat-Dabade that is true, but you are adding multiple args to the event emitter, which is weird.. even emitter should have the name of the event and only one more argument that can be an object as far as I can remember

Reply:
@enahum after multiple attempts, this was the only thing that worked. I checked it with Daniel as well before finalizing this and multiple arguments are supported everywhere other than one place, and even there its just a bad type definition. The tests are there for the component using it. WE can add tests for this specific thing later as part of follow up ticket.

Reply:
@enahum @larkox  filed https://mattermost.atlassian.net/browse/MM-63457 for follow up tests after this is merged to master and QA is in progress. We need to get this thing shipped at some point.

Reply:
This is needed only for tablets BTW.

Reply:
> @enahum @larkox  filed https://mattermost.atlassian.net/browse/MM-63457 for follow up tests after this is merged to master and QA is in progress. We need to get this thing shipped at some point.

Yes is true, this needs to get shipped at some point, that does not mean that is not going to be reviewed with the same level of quality control that it deserves nor means that has to be merged where there are unanswered questions or raised concern. 

I do understand the pressure, and I'm sorry, but if there are things in the PR that I think should be addressed I'll mention it and even block the merge if needed regardless of the pressure as the pressure that builds after is shipped is even bigger.

Hope you take this with the positivity it is given